spec/requests/activity_insight_oa_workflow/wrong_file_version_curation/email_author_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'requests/requests_spec_helper'
+require 'support/authentication'
+
+describe ActivityInsightOAWorkflow::WrongFileVersionCurationController do
+  describe 'POST /activity_insight_oa_workflow/wrong_file_version_curation/email_author' do
+    let(:user) { create(:user, is_admin: true) }
+
+    before do
+      sign_in_as(user)
+      post user_azure_oauth_omniauth_authorize_path
+    end
```

Comment:
We can leave this for another time, but just a note here to say that we might need to handle authentication in same way that we are in the existing controller specs - just by totally mocking out the entire authentication mechanism down at the rack middleware level.

i.e. https://github.com/psu-libraries/researcher-metadata/blob/fac721ad2a7452a17fd0d0421f59693fb96c5dfb/spec/component/controllers/activity_insight_oa_workflow_controller_spec.rb#L16

Reply:
FWIW I was able to get this to work by adding:
```
RSpec.configure do |config|
  config.include Devise::Test::IntegrationHelpers, type: :request
end
```
to the `requests_spec_helper.rb` and then just using `sign_in(user)` rather than:
```
 before do
      sign_in_as(user)
      post user_azure_oauth_omniauth_authorize_path
 end
```

Reply:
Oh, awesome! Thanks, @ajkiessl!

`sign_in(user)` looked like it should have worked - I just didn't realize that I was missing the necessary configuration.